### PR TITLE
Refresh README

### PR DESCRIPTION
**Pull request by achamayou**
Pre-req for #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,17 +94,9 @@ That's it! The next time you create a Pull Request, your CI will automatically s
 
 ## Caveats
 
-- If the CI has never run on the target branch (e.g. `master` - likely to happen when you first set up `cimetrics`), the report will show the metrics results of the branch to merge against itself. It will be displayed fine once the CI has run on the target branch.
+- If the CI has never run on the target branch (e.g. `master` - likely to happen when you first set up `cimetrics`), the report will only show the values that have been uploaded, without any comparison.
 - The rendered images are currently hosted in the target GitHub repository itself, under the `cimetrics` branch, in the `cimetrics` directory.
 
-## to-do
```
Comment (achamayou) [inline README.md:104]: Replaced by issues.

**Comment by ghost** [comment]
readme@[366 aka 20200111.4](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=366&view=results) vs master ewma over 40 builds from [129](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=129&view=results) to [346](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=346&view=results)
images

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 [![Build Status](https://dev.azure.com/jumaffre/metrics-devops/_apis/build/status/jumaffre.metrics-devops?branchName=master)](https://dev.azure.com/jumaffre/metrics-devops/_build/latest?definitionId=1&branchName=master)
 [![PyPI version](https://badge.fury.io/py/cimetrics.svg)](https://badge.fury.io/py/cimetrics)
 
-`cimetrics` lets you track crucial metrics to avoid unwanted regressions. It is easy to integrate with your existing projects and automatically provides quick feedback in your GitHub Pull Requests. See it in action [here](https://github.com/jumaffre/cimetrics/pull/28#issuecomment-519552043). 
+`cimetrics` lets you track crucial metrics to avoid unwanted regressions. It is easy to integrate with your existing projects and automatically provides quick feedback in your GitHub Pull Requests. See it in action [here](https://github.com/jumaffre/cimetrics/issue_comments/573228086).
```
Comment (letmaik) [inline README.md]: Broken url

Comment (achamayou) [reply README.md]: How strange, I must have botched the copy paste... Thank you for catching that.

**Comment by ghost** [comment]
readme@[367 aka 20200111.5](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=367&view=results) vs master ewma over 40 builds from [129](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=129&view=results) to [346](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=346&view=results)
images

File: README.md [hunk 3]
```diff
@@ -3,9 +3,9 @@
 [![Build Status](https://dev.azure.com/jumaffre/metrics-devops/_apis/build/status/jumaffre.metrics-devops?branchName=master)](https://dev.azure.com/jumaffre/metrics-devops/_build/latest?definitionId=1&branchName=master)
 [![PyPI version](https://badge.fury.io/py/cimetrics.svg)](https://badge.fury.io/py/cimetrics)
 
-`cimetrics` lets you track crucial metrics to avoid unwanted regressions. It is easy to integrate with your existing projects and automatically provides quick feedback in your GitHub Pull Requests. See it in action [here](https://github.com/jumaffre/cimetrics/pull/28#issuecomment-519552043). 
+`cimetrics` lets you track crucial metrics to avoid unwanted regressions. It is easy to integrate with your existing projects and automatically provides quick feedback in your GitHub Pull Requests. See it in action [here](https://github.com/jumaffre/cimetrics/pull/66#issuecomment-573324351).
```
Comment (jumaffre) [inline README.md:6]: Should this be a PR that we always keep open on the repo?

File: README.md [hunk 4]
```diff
@@ -38,17 +38,13 @@ You can use the simple python API to push your metrics to your storage. First, m
...
-
-# Run some tests and collect some data
-
-metrics.put("metric1 name (unit)", metric_1)
-metrics.put("metric2 name (unit)", metric_2)
-
-metrics.publish()
+with cimetrics.upload.Metrics() as metrics:
+  # Run some tests and collect some data
+  metrics.put("metric1 name (unit)", metric_1)
```
Comment (jumaffre) [inline README.md:43]: We should show the "^" behaviour here.

File: README.md (hunk 3 above)
Comment (achamayou) [reply README.md:6]: Does it matter if it's open? It seems more reasonable to close it if anything (avoid it being super far behind with a crazy diff and all that)

File: README.md (hunk 4 above)
Comment (achamayou) [reply README.md:43]: I'm hoping to kill that soon and make it a property, along with the category (unit/end-to-end), that's why I've not mentioned it here.

**Pull request merged**